woocommerce/woocommerce-admin
# Remove unreleased section

This PR removes the unreleased section in [changelog.txt](https://github.com/woocommerce/woocommerce-admin/blob/main/changelog.txt) to comply with the changelogger format.


### Detailed test instructions:

1. Run `npm run changelogger -- add` to add a test changelog entry
2. Confirm `npm run changelogger -- write`  command works without an error.

no changelog needed
